### DISCUSSION TITLE
How do you use the People page?

### DISCUSSION THREAD

**Discussion by rmathew1992** [community/community#40472] [Repositories]
### Select Topic Area

Product Feedback

### Body

👋 We'd love to know how you use the People page on the Repository Insights tab (navigate to https://github.com/[REPO_NAME]/people), or click on "Insights" < "People" from your repository.

<img width="1613" alt="Screen Shot 2022-11-30 at 10 28 43 AM" src="https://user-images.githubusercontent.com/8298818/203331070-8a769f76-f967-494e-885e-faf1b309f567.png">
<img width="1300" alt="Screen Shot 2022-11-30 at 1 27 27 PM" src="https://user-images.githubusercontent.com/2595311/204878968-db162c68-00a4-4f01-9b17-3e1416063fbc.png">

**Feedback we'd love to hear from you**

- How often do you use this page?
- What data are you looking to get from it?
- What actions do you expect to take on this page?
- What could we do to make this page easier to use or better for you?
- Anything else you think we should know?

**Comment by mastercoms** [comment]
Honestly, I never knew this page existed. I accidentally clicked on it today and noticed this feedback prompt. I always use the `Settings > Access > Collaborators and teams` page.

**Comment by jlord** [comment]
Thanks for the feedback here! ✨
